src/Components/Preloaded/Preload.tsx

```diff
@@ -0,0 +1,13 @@
+import { FC } from "react";
+import { Spin } from "antd";
+import "antd/dist/antd.css";
```

Comment:
use import "antd/dist/antd.css"; in your main index.js file

Reply:
done